testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -529,6 +529,36 @@ public void testMaterializedView()
         assertUpdate("DROP MATERIALIZED VIEW " + viewNameWithComment);
     }
 
+    @Test
+    public void testViewAndMaterializedViewTogether()
```

Comment:
Looking at this test now, i am not sure what it adds above dedicated test for views and materialized views.

Keep it, if you find is useful, but then maybe worth commenting why it's useful

Reply:
Yeah - I guess it is not useful. I did not see `TestIcebergMetadataListing` which covers that.

Reply:
OH maybe not actually - It adds verification if we can actually query from view in existence of materialized view and vice versa.
Not sure if that is helpful. But adds some coverage....

Reply:
I added
```
// Validate that it is possible to have views and materialized views defined at the same time and both are operational
```
I do not feel strongly we need that. But it is not much cost I think (it only applies to Iceberg)